content/blog/2024/grid-masonry-nuance.md

```diff
@@ -0,0 +1,480 @@
+---
+title: Choosing a masonry syntax in CSS
```

Comment:
```suggestion
title: Choosing a Masonry Syntax in CSS
```
based on: https://deploy-preview-709--oddleventy.netlify.app/styleguide/writing#headlines

I didn't add this suggestion for every headline on this post because it is more obvious when the post titles aren't matching in the list view, but we probably should figure out if we want to stick with that Title Case guideline, or update it to Sentence case ... either way we go is fine with me (knowing we've been a bit inconsistent so older posts will go back and forth). Maybe it is something we should consider at retreat.